lib/android_build/app/src/main/java/com/microsoft/applications/events/maesdktest/MaeUnitLogger.java

```diff
@@ -4,9 +4,6 @@
 //
 package com.microsoft.applications.events.maesdktest;
 
-import androidx.annotation.Keep;
```

Comment:
Wasn't this necessary to avoid the obfuscator from stripping/changing the method signature?

Reply:
This is app that executes tests, is not part of maesdk